### PR TITLE
Add seperator between @param's name and description for better readability.

### PR DESCRIPTION
**Pull request by aars**

### CODE REVIEW

File: autoload/jsdoc.vim [hunk 1]
```diff
@@ -50,6 +50,10 @@ endif
 if !exists('g:jsdoc_allow_shorthand')
   let g:jsdoc_allow_shorthand = 0
 endif
+" Use seperator between @param name and description.
+if !exists('g:jsdoc_param_description_seperator')
+	let g:jsdoc_param_description_seperator = " - "
```
Comment (heavenshell) [inline autoload/jsdoc.vim]: Could you change default behavior?
I don't like bc break.
`jsdoc_param_description_seperator` should be `""`

**Comment by ryanoasis** [comment]
:+1: For not changing the default that it is currently but I do like the change of being configurable :smile:

**Comment by aars** [comment]
@heavenshell Good call. Reset the default.

**Comment by heavenshell** [comment]
Merged!
Thank you for PR :+1:

**Comment by aars** [comment]
NP. Thank you for the plugin! Cheers!

**Pull request merged**